docs/index.md

```diff
@@ -24,38 +24,24 @@ See the relationship between the connector files (in blue) and the Tableau **Con
 
 ![]({{ site.baseurl }}/assets/files-overview.png)
 
-## Before you begin
+# What is a Taco?
+A `.taco` file is a packaged Tableau connector file that can be dropped into your `My Tableau Repository/Connectors` folder. They will be automatically loaded by Tableau.
+
+For more information about packaging your connector into a Taco, refer to [How to Package and Sign Your Connector for Distribution]({{ site.baseurl }}/docs/package-sign)
+
+# Before you Begin
 
 You need to do these things before you start:
 
 - Install the ODBC or JDBC driver that you’ll use with the connector you’ll create.
 - Install Tableau Desktop 2019.2 or later on a Windows or Mac computer.
```

Comment:
Since we mention 2019.2 here, would it be useful to have a section that describes what you can do with 2019.2 vs 2019.4+, or just make explicit that the "Developer Path" option is what you can do on 2019.2/.3?